shared/base/view.js

```diff
@@ -258,6 +258,15 @@ module.exports = BaseView = Backbone.View.extend({
       };
     }
 
+    // Allow ability to just pass the full "spec" to a lazy loaded view
+    if (this.options.fetch_spec) {
```

Comment:
Also, if you look at the rest of the fetch params and model name they're all snake case because of that.